modules/energid/charts.go

```diff
@@ -0,0 +1,108 @@
+package energid
+
+import "github.com/netdata/go.d.plugin/agent/module"
+
+type (
+	Charts = module.Charts
+	Dims   = module.Dims
+)
+
+var charts = Charts{
+	{
+		ID:    "blockindex",
+		Title: "Blockchain Index",
+		Units: "count",
+		Fam:   "blockchain",
+		Ctx:   "energid.blockindex",
+		Type:  module.Area,
+		Dims: Dims{
+			{ID: "blockchain_blocks", Name: "Blocks", Algo: module.Absolute},
+			{ID: "blockchain_headers", Name: "Headers", Algo: module.Absolute},
+		},
+	},
+	{
+		ID:    "difficulty",
+		Title: "Blockchain Difficulty",
+		Units: "difficulty",
+		Fam:   "blockchain",
+		Ctx:   "energid.difficulty",
+		Type:  module.Line,
+		Dims: Dims{
+			{ID: "blockchain_difficulty", Name: "Diff", Algo: module.Absolute},
+		},
+	},
+	{
+		ID:    "mempool",
+		Title: "MemPool",
+		Units: "MiB",
+		Fam:   "memory",
+		Ctx:   "energid.mempool",
+		Type:  module.Area,
+		Dims: Dims{
+			{ID: "mempool_max", Name: "Max", Algo: module.Absolute, Div: 1024 * 1024},
+			{ID: "mempool_current", Name: "Usage", Algo: module.Absolute, Div: 1024 * 1024},
+			{ID: "mempool_txsize", Name: "TX Size", Algo: module.Absolute, Div: 1024 * 1024},
+		},
+	},
+	/*
+		We had this chart on Python version, but it is not documented at
+		https://github.com/energicryptocurrency/core-api-documentation
+		{
+			ID:    "secmem",
+			Title: "Secure Memory",
+			Units: "KiB",
+			Fam:   "memory",
+			Ctx:   "energid.secmem",
+			Type:  module.Area,
+			Dims: module.Dims{
+				{ID: "secmem_total", Name: "Total", Algo: module.Absolute},
+				{ID: "secmem_locked", Name: "Locked", Algo: module.Absolute},
+				{ID: "secmem_used", Name: "Used", Algo: module.Absolute},
+			},
```

Comment:
respone has no `secmem_*`? if so just delete it

Reply:
We do not have it more running on `2.4.x`, I believe it was removed. I removed it.